test/parallel/test-http2-util-nghttp2error.js

```diff
@@ -14,3 +14,9 @@ common.expectsError(() => {
   type: NghttpError,
   message: 'Invalid argument'
 });
+
+// Should convert the NghttpError object to string properly
+{
+  const err = new NghttpError(401);
+  strictEqual(err.toString(), 'Error [ERR_HTTP2_ERROR]: Unknown error code');
```

Comment:
Emmm, wondering why we need to converting to string?

Reply:
I think maybe for logging purpose.

Reply:
@ZYSzys this is done implicitly when testing an error against regular expressions or when added to a string and more. This is often done and this PR konly verifies that our testing function does what it should (internal errors are special due to the `code` property being visible after the error name).